app/javascript/components/explore/ExploreDisplayPanelManager.jsx

```diff
@@ -492,13 +497,24 @@ export default function ExploreDisplayPanelManager({
               exploreParams={exploreParamsWithDefaults}
               updateExploreParams={updateExploreParams}
               allGenes={exploreInfo ? exploreInfo.uniqueGenes : []}/>
-            {exploreInfo?.bookmarks &&
-              <BookmarkManager
-                bookmarks={exploreInfo.bookmarks}
-                studyAccession={studyAccession}
-                clearExploreParams={clearExploreParams}/>
-            }
-
+            <div id='bookmark-container'>
+              <button className="action action-with-bg"
+                      onClick={clearExploreParams}
+                      title="Reset all view options"
+                      data-analytics-name="explore-view-options-reset">
+                <FontAwesomeIcon icon={faUndo}/> Reset view</button>
+              <button onClick={copyLink}
+                      className="action action-with-bg"
+                      data-toggle="tooltip"
+                      title="Copy a link to this visualization to the clipboard">
+                <FontAwesomeIcon icon={faLink}/> Get link</button>
+              {exploreInfo?.bookmarks &&
+                <BookmarkManager
+                  bookmarks={exploreInfo.bookmarks}
+                  studyAccession={studyAccession}
+                  clearExploreParams={clearExploreParams}/>
```

Comment:
Potential lurking bug: `clearExploreParams` is no longer a parameter in the `BookmarkManager` function.

Reply:
Nice catch!